docs/authentication/embedded-sdk-add-passkeys.mdx

```diff
@@ -118,16 +130,32 @@ While app schemes are generally easier to set up, Universal URLs and App Links a
   </mark>
 </p>
 
+  <p>
+    <font color="green">WM: MARK</font>
+  </p>
+
 Calling the token endpoint is the second step in the authorization flow and usually happens in your backend if your application's Client Type is `Confidential` <mark>What if the Client Type is **Public**?</mark>. Make sure to a call the authorization endpoint first to retrieve an authorization code.
 
+  <p>
+    <font color="green">WM: MARK</font>
+  </p>
+
 <p>
   <mark>
     Are they parsing the <code>redirectUrl</code> returned when calling the function <code>Embedded.authenticate</code> for a code in the query parameters and then exchanging that code for an access token?
```

Comment:
Correct. We loosely reference this in the code snippets when we say:

```kotlin
// This URL contains authorization code and state parameters
// Exchange the authorization code for an id_token using Beyond Identity's token endpoint.
var code = parseCode(redirectUrl)
var token = exchangeForToken(code)
```

How should we make this clearer? Is showing a sample redirectUrl with the parameters sufficient?